app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> "Person":
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
+            Person.people[person["name"]].wife = (
+                Person.people)[person["wife"]]
+            continue
```

Comment:
fix formatting
[use this checklist](https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md)

 `braces must be located in one of two options: open and start with the text or have a line break between the text`